membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -350,14 +336,12 @@ object SupporterRatePlanToAttributesMapper {
 sealed abstract class MembershipTier(val name: String, val value: Int)
 
 object MembershipTier {
-  case object Friend extends MembershipTier("Friend", 1)
-  case object Staff extends MembershipTier("Staff", 2)
   case object Supporter extends MembershipTier("Supporter", 3)
   case object Partner extends MembershipTier("Partner", 4)
   case object Patron extends MembershipTier("Patron", 5)
```

Comment:
wasn't sure whether to renumber the ordering but I don't think It'll hurt to leave it